Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml

```diff
@@ -231,13 +231,54 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: d689ef6d-071c-4105-8c81-49840217f8ae
+    type: regular
+    task:
+      id: d689ef6d-071c-4105-8c81-49840217f8ae
+      version: -1
+      name: Set attack surface rule fields
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setIncident
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
```

Comment:
Please use selectors here: https://xsoar.pan.dev/docs/playbooks/playbook-conventions

Reply:
Selectors are not allowed any more in the new version of XSOAR for simple selectors.